commands/dev-deploy.js

```diff
@@ -50,7 +50,7 @@ async function devDeploy(options) {
     const result = await account.deployContract(contractData);
     inspectResponse.prettyPrintResponse(result, options);
     console.log(`Done deploying to ${accountId}`);
-};
+}
```

Comment:
nit I think we want the semicolons for fns for consistency. I wonder why the linter didn't catch this.

Reply:
actually the linter told me the opposite in this case and that's why I removed it :)

Reply:
heh, why wasn't it complaining before? go home linter, you are drunk. Ok, adding this to a box of unexplained mysteries and closing the thread.